app/javascript/components/close_contact/CloseContact.js

```diff
@@ -281,7 +283,7 @@ class CloseContact extends React.Component {
           {/* This does not impact component functionality at all. */}
           {this.state.disableCreate && (
             <ReactTooltip id="create-tooltip" multiline={true} place="top" type="dark" effect="solid" className="tooltip-container text-left">
-              Please enter one of the following: First Name, Last Name, Phone Number, or Email
+              Please enter a name (First Name or Last Name) and a reporting method (Phone Number or Email).
```

Comment:
I would add "at least one name (first or last) and at least one reporting method (phone or email) just to be extremely clear about this

Reply:
I ran it by a member of the PO Team, and they were okay with the tooltip wording. @kylietmo do you have any opinions here? I'm happy to change it, would just like a second opinion

Reply:
I agree with Sarah on this one - can only help to be super explicit in this case.

Reply:
Can we also make this say "contact method" instead of "reporting method"? Close contacts aren't reporting until they're enrolled.

Reply:
I asked a member of the PO team about that, as I initially wrote `Contact Method` and she wrote back `"One minor item--can we change "contact method" to "reporting method"? That will keep it consistent with the language in the "Monitoree Contact Information" section that those fields will populate upon enrollment."`

Reply:
Okay, that's fine. I think the change Sarah mentioned should be fine though.